src/explore-education-statistics-frontend/src/modules/download/components/TopicDownloadList.tsx

```diff
@@ -31,22 +32,40 @@ function TopicDownloadList({ topic }: Props) {
             </p>
 
             <ul data-testid={title}>
-              {downloadFiles.map(({ extension, name, path, size }) => (
-                <li key={path}>
-                  <Link
-                    to={`${process.env.CONTENT_API_BASE_URL}/download/${path}`}
-                    data-testid={`download-stats-${path}`}
-                    analytics={{
-                      category: 'Downloads',
-                      action: `Download latest data page ${name} file downloaded`,
-                      label: `File URL: /api/download/${path}`,
-                    }}
-                  >
-                    {name}
-                  </Link>
-                  {` (${extension}, ${size})`}
-                </li>
-              ))}
+              {downloadFiles.map(
+                ({ id: fileId, fileName, extension, name, size }) =>
+                  !fileId && name === 'All files' ? (
+                    <li key={fileName}>
```

Comment:
Wouldn't `fileId` be better?

Reply:
That's sorted with the other suggestion now, thanks.